atd-etl/cris_import/cris_import.py

```diff
@@ -60,6 +62,8 @@ def main():
     global GRAPHQL_ENDPOINT_KEY
     global SFTP_ENDPOINT_SSH_PRIVATE_KEY
```

Comment:
it looks like perhaps all three of these variables may no longer be in use

Reply:
The graphql endpoint ones need to persist, but the SFTP endpoint key is outta here!